### PR TITLE
Move newton into package:flutter

### LINKED ISSUE #2441: Move newton into flutter package
~~I believe @abarth has this underway.~~

### PR DESCRIPTION
**Pull request by abarth**
Rather that importing `package:newton/newton.dart` you can
`import package:flutter/physics.dart`.

Fixes #2441

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde @HansMuller

**Comment by chinmaygarde** [comment]
lgtm

File: packages/flutter/lib/physics.dart [hunk 1]
```diff
@@ -4,17 +4,17 @@
 
 /// Simple one-dimensional physics simulations, such as springs, friction, and
 /// gravity, for use in user interface animations.
-/// 
+///
 /// This library is not meant to be imported by developers.
```
Comment (pylaligand) [inline packages/flutter/lib/physics.dart]: Looks like this comment is now obsolete.

Comment (abarth) [inline packages/flutter/lib/physics.dart]: Yep!  Thanks.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,9 +78,9 @@ Individual tests can also be run directly, e.g. `flutter test lib/my_app_test.da
 
 Flutter tests use [package:flutter_test](https://github.com/flutter/flutter/tree/master/packages/flutter_test) which provides flutter-specific extensions on top of [package:test](https://pub.dartlang.org/packages/test).
 
-`flutter test` runs tests inside the flutter shell.  Some packages inside the flutter repository can be run inside the dart command line VM as well as the flutter shell, `packages/newton` and `packages/flutter_tools` are two such examples:
+`flutter test` runs tests inside the flutter shell.  Some packages inside the flutter repository can be run inside the dart command line VM as well as the flutter shell, `packages/flutter_tools` is one such examples:
```
Comment (Hixie) [inline CONTRIBUTING.md]: one such example"s"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -78,9 +78,9 @@ Individual tests can also be run directly, e.g. `flutter test lib/my_app_test.da
...
 
- * `cd packages/newton`
+ * `cd packages/flutter_tools`
  * `pub run test`
```
Comment (Hixie) [inline CONTRIBUTING.md]: we actually don't run the tests that way any more btw. we use dart to run an all.dart file directly.

**Pull request merged**